website/content/en/docs/upgrading-sdk-version/v1.16.0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: v1.16.0
+weight: 998984000
+---
+
+## Header text for the migration section
+
+Add into the config/manager/manager.yaml` the following limits scaffold by default:
+```yaml ... # TODO(user): Configure the resources accordingly based on the project requirements. # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ resources:
+  limits:
+    cpu: 500m
+    memory: 768Mi
+  requests:
+    cpu: 10m
+    memory: 256Mi
+... ```
```

Comment:
Formatting nits:
```suggestion
Add into the `config/manager/manager.yaml` the following limits which are now scaffolded by default:
```yaml
 ... 
 # TODO(user): Configure the resources accordingly based on the project requirements. # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ resources:
  limits:
    cpu: 500m
    memory: 768Mi
  requests:
    cpu: 10m
    memory: 256Mi```
```